gym_car_race/SelfDriveEnv.py

```diff
@@ -375,7 +375,7 @@ def __init__(self, config=cfg):
         self.height = config["car"]['height']
         self.image = pygame.transform.scale(self.image, (self.width, self.height))        
         self.angle = config["car"]['angle']
-        self.pos = config["car"]["position"]
+        self.pos = [100, 100]
```

Comment:
Instead of setting the position to some arbitrary value like [100, 100], we can simply just wait for the user to draw a map and set a start position that way before rendering the car. Let's set the initial position to `None`

Reply:
Do we need to also add the code to calculate the start position of care according to the track map?